cli/cmd/create_project.go

```diff
@@ -139,29 +139,48 @@ keptn create project PROJECTNAME --shipyard=FILEPATH --git-user=GIT_USER --git-r
 				return errors.New("Proxy cannot be set without scheme")
 			}
 
-			project.GitProxyURL = *createProjectParams.GitProxyURL
-			project.GitProxyScheme = *createProjectParams.GitProxyScheme
-			project.GitProxyUser = *createProjectParams.GitProxyUser
-			project.GitProxyPassword = *createProjectParams.GitProxyPassword
-			project.InsecureSkipTLS = *createProjectParams.InsecureSkipTLS
+			project.GitCredentials = &apimodels.GitAuthCredentials{
+				User:      *createProjectParams.GitUser,
+				RemoteURL: *createProjectParams.RemoteURL,
+			}
 
 			if strings.HasPrefix(*createProjectParams.RemoteURL, "ssh://") {
 				content, err := ioutil.ReadFile(*createProjectParams.GitPrivateKey)
 				if err != nil {
 					return fmt.Errorf("unable to read privateKey file: %s\n", err.Error())
 				}
 
-				project.GitPrivateKey = string(base64.StdEncoding.EncodeToString(content))
-				project.GitPrivateKeyPass = *createProjectParams.GitPrivateKeyPass
-			}
+				sshCredentials := apimodels.SshGitAuth{
+					PrivateKey:     string(base64.StdEncoding.EncodeToString(content)),
```

Comment:
the `string()` conversion can be removed since `EncodeToString` already returns a `string`